optimum/habana/fx/optimization/transformations.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+import torch
+from transformers.activations import GELUActivation
+
+from optimum.fx.optimization import ReversibleTransformation
+
+from ..tracer import GELU_CLASSES_TO_TRACK
+
+
+if TYPE_CHECKING:
+    from torch.fx import GraphModule
+
+
+class GeluToFusedGelu(ReversibleTransformation):
+    """
+    Transformation that replaces GELU modules by `torch.nn.functional.gelu` (i.e. `GELUActivation()`) if different.
+    `torch.nn.functional.gelu` is mapped to a single fused GELU kernel on Gaudi so the transformed model will be faster.
+    """
+
+    def transform(self, graph_module: "GraphModule") -> "GraphModule":
+        self.original_gelu_module = None
+        for node in graph_module.graph.nodes:
+            if node.op == "call_module":
```

Comment:
I think `torch.nn.functional.gelu` is more of a `call_function` node?

So you could check by doing:

```python
if node.op == "call_function" and node.target is torch.nn.functional.gelu:
```